cmd/share.go

```diff
@@ -63,13 +65,20 @@ var shareCmd = &cobra.Command{
 			fmt.Println("Share revoked for client " + refereeClientID)
 		} else {
 			expiration, _ := cmd.Flags().GetInt64("expiration-seconds")
+			available, _ = cmd.Flags().GetInt64("available-after")
+
+			if len(refereeClientID) == 0 && available > 0 {
+				PrintError("Cannot set available-after if the clientid is empty")
```

Comment:
@Rizary Can we actually support empty clientID?

Reply:
@Rizary  it means public share if clientID is not passed. it is mentioned on README.md

Reply:
Yes, I'm aware of the public share. In this case, if it is public share, then they can't pass the availability-after option. Am I correct? I think this is the design decision on the available-after.

Reply:
I think public share should have the feature too. cc @moldis